cmd/prepare-storage.go

```diff
@@ -178,8 +178,11 @@ func prepForInitXL(firstDisk bool, sErrs []error, diskCount int) InitActions {
 		if disksFormatted+disksOffline == diskCount {
 			return InitObjectLayer
 		}
-		// Some of the formatted disks are possibly corrupted or unformatted, heal them.
-		return WaitForHeal
+		// We need write quorum for heal.
+		if disksFormatted >= quorum+1 {
```

Comment:
For reasons explained in the previous comment, this would be,
```go
if disksFormatted >= quorum {
  return WaitForHeal
}
```
where ``quorum = diskCount/2``.